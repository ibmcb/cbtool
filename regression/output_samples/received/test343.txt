 status: Multiple samples of management metrics for all HOSTs were found (the file /home/msilva/cloudbench/lib/auxiliary//../../data/REGRESSION_B/HOST_management_REGRESSION_B.csv will *NOT* be empty).
Monitor extraction success. HOST management performance data samples were written to the file /home/msilva/cloudbench/lib/auxiliary//../../data/REGRESSION_B/HOST_management_REGRESSION_B.csv
running shell command: "cat ~/cloudbench/data/REGRESSION_B/HOST*man*REGRESSION*"....
stdout:
 #field:column #
#uuid:1
#name:2
#cloud_ip:3
#model:4
#vmc_name:5
#vmc_cloud_ip:6
#mgt_001_provisioning_request_originated:7
#mgt_002_provisioning_request_sent:8
#mgt_003_provisioning_request_completed:9
#mgt_004_network_acessible:10
#mgt_005_file_transfer:11
#mgt_006_application_start:12
#mgt_901_deprovisioning_request_originated:13
#mgt_902_deprovisioning_request_sent:14
#mgt_903_deprovisioning_request_completed:15

uuid,name,cloud_ip,model,vmc_name,vmc_cloud_ip,mgt_001_provisioning_request_originated,mgt_002_provisioning_request_sent,mgt_003_provisioning_request_completed,mgt_004_network_acessible,mgt_005_file_transfer,mgt_006_application_start,mgt_901_deprovisioning_request_originated,mgt_902_deprovisioning_request_sent,mgt_903_deprovisioning_request_completed
EAE8FC54-444A-5B98-99C5-3CCEF3ECDA7B,host_simhosta1,184.207.211.178,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
2C8D53FF-39B5-51CA-BF16-EEF11442B76E,host_simhosta2,12.113.14.5,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
DF11F04C-A233-505A-8F7F-FB65C8C36634,host_simhosta3,196.228.140.246,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
0EBAD52A-EEC5-5B0B-8CA0-C709879D0939,host_simhosta4,204.138.187.150,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
451FAD6C-E701-54AA-9C25-F0AB2A36CC2F,host_simhostb0,92.148.156.4,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
24E27D04-06EA-5BDE-866C-FC16AEF3D837,host_simhostb4,7.72.194.46,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
F5AD202C-E1BB-5020-8925-6918D7749BCA,host_simhostc2,199.174.107.241,sim,NA,NA,18,0,0,NA,NA,NA,NA,NA,NA
A0327700-DEC2-584E-A4DA-C59716D65C34,host_simhostc3,227.5.35.220,sim,NA,NA,18,0,0,NA,NA,NA,NA,NA,NA
2401DF64-14C9-50FD-ADE5-462F542A2E09,host_simhostc4,95.69.220.78,sim,NA,NA,18,0,0,NA,NA,NA,NA,NA,NA
361F85A4-3381-5C6C-B6F9-B3507C540C9D,host_simhostd1,78.73.202.51,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
8FEE8FFE-5D81-5D9A-9B3F-EBA98DB35E7E,host_simhostd2,17.80.17.75,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
5DA54FD8-86DA-56DD-B1DD-2593F56B1A30,host_simhostd0,11.76.151.145,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
A741039A-7AE3-50EC-B01D-AB13CE8C796C,host_simhostd3,233.227.53.21,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
2FEC0372-DEC0-55D2-83C3-5ABD985F15EB,host_simhostd4,48.150.195.214,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
3ECC6057-08E5-525A-B4C1-C3B5288B9952,host_simhostc0,112.83.28.155,sim,NA,NA,18,0,0,NA,NA,NA,NA,NA,NA
54B64C75-2DF7-5EFF-B115-C2F10EA06556,host_simhostc1,194.46.159.28,sim,NA,NA,18,0,0,NA,NA,NA,NA,NA,NA
72ADD023-2A15-56CE-AE03-675E0110E382,host_simhostb1,57.173.240.103,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
0C3BAACF-A555-59B9-948F-993696F94340,host_simhostb2,10.23.129.163,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
5770672C-10A1-5CB0-8604-4ACC9F3B785B,host_simhostb3,37.133.116.170,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
C5A63153-A27E-5F01-8DDE-C0FA42717830,host_simhosta0,71.227.154.98,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
8D493224-C3FD-5F73-BB2E-62C664B3664E,host_simhostb0,134.184.134.217,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
093712D5-AA79-5C7F-A0D7-B6596DF48D24,host_simhostc0,210.23.235.55,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
796C5EC3-0932-50CF-90A1-A2197B601B8E,host_simhosta0,165.78.232.248,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
9F97A022-AAEA-587F-B1CF-4F6896966A84,host_simhostd0,104.204.12.32,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
D6ED860B-1C7A-5390-B109-54F79F8994CF,host_simhostc1,36.48.115.156,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
59963D19-2ADC-5283-8E60-D5B117B79363,host_simhostb1,168.2.19.76,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
AE553BE4-1DE8-574B-A65B-634007A9B228,host_simhostd1,247.174.254.30,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
8EBD041B-B01D-5F5D-B699-2EE0D2CCA495,host_simhosta1,245.170.242.52,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
C04E6DFB-1E7D-501F-92C2-83244F5D6DDF,host_simhostc2,186.218.252.139,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
2D05DF0A-2917-58BA-98E5-050C88278314,host_simhostd2,47.244.170.190,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
DF8894E7-FCA3-52EE-BB62-CE82285CFEEE,host_simhostb2,85.46.115.41,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
8093EA56-8C5C-5A61-901E-66DC13E07B19,host_simhosta2,23.188.103.224,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
0A5CEAA4-BB77-5966-A1B9-EC5EA42BE654,host_simhostc3,8.5.210.33,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
E7A98668-DF1F-56CE-9FE4-D37EAB9E8254,host_simhostb3,70.36.189.223,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
31AF253D-9B01-58E6-8A36-AB4302AF5355,host_simhostd3,255.21.61.119,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
53AD1CAE-86FF-5980-8894-E18A10E0B060,host_simhosta3,227.3.139.2,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
5C4158C6-B3C3-58E3-B60C-5C124CB3DA74,host_simhostc4,78.245.196.127,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
63FF69D7-FC04-5A56-A40F-9B691CA4C064,host_simhostb4,130.191.226.127,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
A025350B-011B-50F3-9D12-7A13E5E282D3,host_simhostd4,19.59.209.172,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
B42F37F9-DBE6-508B-B5BD-188F2DCE38AE,host_simhosta4,103.104.100.152,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
2E53F74A-041A-53BA-A95C-277DF58AC0A8,host_simhosta0,187.6.85.188,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
1D1E584D-7FFB-5F4F-B06C-1CB3EFF251CD,host_simhostb0,69.57.121.224,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
5E126B12-17D3-5F7C-A68D-C3800F0792F3,host_simhostc0,89.18.100.204,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
F54108D2-D396-529A-B23E-A7D7D1DB18A8,host_simhostd0,81.61.106.205,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
1EFFB64A-6F6C-57DE-B8E6-4E336FFF12A8,host_simhosta1,157.69.5.246,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
E252AFD4-8101-5943-96D3-0EC2CEB5B4CD,host_simhostb1,224.171.88.78,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
589687C8-7928-5BFF-AD2C-D05D2C4D4259,host_simhostc1,222.201.228.175,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
C7FB09EC-4C24-5035-A47D-818F24A75805,host_simhostd1,135.137.127.152,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
5B96D443-FC87-5D31-86F1-AE9C9914E668,host_simhosta2,188.144.101.102,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
5889F64A-668A-566B-BE0A-25D68B2AA789,host_simhostb2,135.146.238.243,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
B57A0B97-FD4B-56A4-BDE3-1C4277253EEA,host_simhostc2,143.140.232.80,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
5A6BBC46-EA80-5BD6-90BC-A103835D7C0C,host_simhostd2,38.161.83.18,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
72302E5E-96E3-51A9-96B7-E83CA02385B5,host_simhosta3,108.145.95.130,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
FD5BA64B-6F18-54E8-B178-8E109D716852,host_simhostb3,160.24.222.15,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
417A64CF-2E97-5DBC-B349-3E74E62E7370,host_simhostc3,243.115.243.172,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
1178F5E1-985E-569C-BDF5-63FAF93904D8,host_simhostd3,69.227.70.235,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
32015123-6519-5764-94BC-F3B46D2B109E,host_simhosta4,245.243.46.142,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
900C72FB-232D-5458-B8D4-7BA9FA501876,host_simhostb4,31.190.54.78,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
2B774948-30D0-5627-9B5B-FFE7FDC17331,host_simhostc4,227.140.135.219,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
A4865CED-4C11-5396-8E7B-137CB98F68C8,host_simhostd4,39.147.212.107,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
02A87729-F726-5F81-8D0D-7F3B61CF358F,host_simhosta0,24.169.15.34,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
CD61FF5A-BFEF-5152-9901-96E671597CD9,host_simhostb0,76.63.10.163,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
41DA94E9-4EDC-543A-B4A4-439D8656A32A,host_simhostd0,157.232.5.229,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
244F60B5-7B7C-51D6-ACDC-435B2900E022,host_simhostc0,179.20.229.230,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
DDBB4065-621B-5FA8-A049-E6C468AD8ED6,host_simhostb1,95.83.132.61,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
6FD71C74-052C-564B-B9F2-90F119EBA70E,host_simhosta1,160.46.76.168,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
F5D7C222-2145-5B56-BADF-959129F02DAB,host_simhostd1,207.82.22.231,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
6CC62DAB-3572-544B-92D8-BE23C4EB2F4A,host_simhostc1,14.94.116.120,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
1FD48A3C-1FAE-57CC-A9CC-963A48CAA9E4,host_simhostb2,139.240.109.118,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
658A1119-DCB4-5982-BDB7-F2C3CA0D51FC,host_simhostd2,184.97.156.191,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
C4901941-3F08-572C-93EC-5CEAB6AD7648,host_simhosta2,220.240.227.169,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
44653BAC-1BD2-57F3-A472-DCE57CF7FD91,host_simhostc2,160.116.76.235,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
C329F64F-C372-5896-A514-B7FE7412317C,host_simhostd3,200.194.32.107,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
E68DDC2C-8D71-5656-A5E0-E1B928F465EE,host_simhostb3,116.57.214.197,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
0C9011CF-9CD2-58AC-A77E-756A7DDDA45B,host_simhosta3,144.137.200.103,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
84E3AE25-457E-5D3C-A9CE-F4DEB020D3D0,host_simhostc3,147.23.39.17,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
E7E3C7DA-B54E-5248-B3E9-492D798C0941,host_simhosta4,98.114.129.145,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
47ECF236-2071-5C8E-A480-CC2C210FF8F6,host_simhostd4,101.60.8.185,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
F5066EF3-2910-5E4A-A46B-38B787F3F44A,host_simhostb4,21.219.64.143,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
CF033F0B-AC3C-5CBE-9790-AC20F4065F37,host_simhostc4,57.234.83.225,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA


