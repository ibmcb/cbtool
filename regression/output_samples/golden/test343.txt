 status: Multiple samples of management metrics for all HOSTs were found (the file /home/msilva/cloudbench/lib/auxiliary//../../data/REGRESSION_B/HOST_management_REGRESSION_B.csv will *NOT* be empty).
Monitor extraction success. HOST management performance data samples were written to the file /home/msilva/cloudbench/lib/auxiliary//../../data/REGRESSION_B/HOST_management_REGRESSION_B.csv
running shell command: "cat ~/cloudbench/data/REGRESSION_B/HOST*man*REGRESSION*"....
stdout:
 #field:column #
#uuid:1
#name:2
#cloud_ip:3
#model:4
#vmc_name:5
#vmc_cloud_ip:6
#mgt_001_provisioning_request_originated:7
#mgt_002_provisioning_request_sent:8
#mgt_003_provisioning_request_completed:9
#mgt_004_network_acessible:10
#mgt_005_file_transfer:11
#mgt_006_application_start:12
#mgt_901_deprovisioning_request_originated:13
#mgt_902_deprovisioning_request_sent:14
#mgt_903_deprovisioning_request_completed:15

uuid,name,cloud_ip,model,vmc_name,vmc_cloud_ip,mgt_001_provisioning_request_originated,mgt_002_provisioning_request_sent,mgt_003_provisioning_request_completed,mgt_004_network_acessible,mgt_005_file_transfer,mgt_006_application_start,mgt_901_deprovisioning_request_originated,mgt_902_deprovisioning_request_sent,mgt_903_deprovisioning_request_completed
D38A5379-C705-5ED9-AC2D-06595D0F6B00,host_simhosta1,210.134.149.154,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
A03FC5D9-2C2F-5E97-BDF2-F38F7ABE35F1,host_simhosta3,92.37.220.26,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
26606CCD-E764-5C48-A008-6C2403B6E1EB,host_simhostc0,191.62.238.4,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
981A7CB1-0691-526D-B4C9-771D3ADCC0E5,host_simhostc1,205.83.31.13,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
5DE66863-90B3-512A-8ED3-3E6A9C4E6C00,host_simhostc2,205.103.174.153,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
3517CB42-68F0-5C2E-8565-17BB19E9805C,host_simhostc4,37.43.241.88,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
DBFA6D88-6E96-5587-8C1F-EEF5CB8EC982,host_simhostd0,41.247.40.244,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
C32ED59A-2927-5C9A-BDC9-ECD6336C4F9B,host_simhostd1,198.191.91.98,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
1575656A-CA00-5015-95E1-B20DAC313927,host_simhostd2,15.185.21.101,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
76076C5D-4A24-5AEA-BFF8-CDD8370F4266,host_simhostd3,129.208.78.221,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
C8BFDFE0-10EF-5CBB-95EB-CAF66D8914C8,host_simhostd4,89.233.162.74,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
21936F89-9D5B-5198-9014-B47C5994AF0B,host_simhostc3,74.140.21.145,sim,NA,NA,19,0,0,NA,NA,NA,NA,NA,NA
C1A45708-C358-5AA9-9C80-C8A06AB259CA,host_simhostb0,114.64.204.49,sim,NA,NA,11,0,0,NA,NA,NA,NA,NA,NA
027DADC2-A514-5212-93F6-8ECA4332179E,host_simhostb1,16.128.190.20,sim,NA,NA,11,0,0,NA,NA,NA,NA,NA,NA
8A9E7AE8-8760-52F8-A464-45D3B875BAA7,host_simhostb2,79.119.247.250,sim,NA,NA,11,0,0,NA,NA,NA,NA,NA,NA
BEBE69B8-2BB8-5A6B-99EC-1FFD22C2E453,host_simhostb3,186.60.121.160,sim,NA,NA,11,0,0,NA,NA,NA,NA,NA,NA
9FE91B14-3912-5F7C-93F4-7550B451525C,host_simhostb4,137.99.109.42,sim,NA,NA,11,0,0,NA,NA,NA,NA,NA,NA
E858C7EA-552A-51F5-A858-B90D20292D12,host_simhosta0,91.91.192.170,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
708B4E39-71ED-5EFE-995A-96D011E5ECFB,host_simhosta2,62.90.188.181,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
541A00D0-29C5-5BE8-ABDF-B91C81592FAF,host_simhosta4,116.217.3.170,sim,NA,NA,10,0,0,NA,NA,NA,NA,NA,NA
D53075F6-2C4B-5CA2-824E-AFF7F5546B9E,host_simhostc0,47.189.165.11,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
DE061038-841A-5A5B-9267-26A0DEFDC446,host_simhosta0,30.216.50.38,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
15FAD2D6-3960-5B7D-8A9F-86512FF43585,host_simhostb0,162.180.251.66,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
ECC89945-432F-54AB-B65E-D97BE40D1F59,host_simhostd0,231.218.18.233,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
F0EAF6E8-5380-5A6B-AFF8-DF1A3F9CC63D,host_simhosta1,92.117.114.78,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
D234E6D7-9983-5995-8610-E73336F62D96,host_simhostc1,141.83.15.65,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
2638820E-9EE4-560D-86B3-FC56BA1CD82C,host_simhostb1,52.81.41.12,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
DA120EEF-0EAF-5433-9F3B-2C7EDBDD6C91,host_simhostd1,166.154.98.213,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
5631DE2E-0801-5D72-A554-AB6344ADA298,host_simhostc2,220.241.122.203,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
7AC1839D-3D3B-5E3B-B8A9-EF1CD0035E18,host_simhosta2,41.48.233.196,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
FE60DBB7-8055-53EF-B52F-0B3EB4AEC2F4,host_simhostb2,91.90.171.136,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
C8C080BA-2779-5BB2-8357-C8D96B8BA463,host_simhostd2,110.228.115.82,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
CB31C6F9-58DE-5908-871F-D84FA1050C6E,host_simhostc3,232.120.221.211,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
E4491A95-655A-5B1A-B871-14C4786786FC,host_simhosta3,176.2.128.130,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
2A605837-20FB-5724-9AD5-083BF87A77E1,host_simhostb3,147.12.160.149,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
28FAC0DF-CE32-51CA-A2BB-E05FB7F73AB3,host_simhostd3,253.213.31.38,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
1509B84C-772D-5075-875A-B2E0A3513777,host_simhostc4,218.17.97.80,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
E6DAB66D-3261-5BC4-AC7A-09300E65A313,host_simhosta4,231.62.115.232,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
BBC4C3C1-6C5F-5593-AC48-29530E0928DA,host_simhostb4,153.44.67.10,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
979C10D2-AC5E-589B-B66E-4B3D76868D1E,host_simhostd4,188.162.143.218,sim,NA,NA,36,0,0,NA,NA,NA,NA,NA,NA
AB375BC0-1821-5375-9AFA-9DBA1661D5AC,host_simhosta0,134.53.213.243,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
A9E9D874-1A57-5A6C-8806-6E4B31A27F36,host_simhostb0,200.189.252.137,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
A6424273-C6E1-5F07-A139-BE1094533464,host_simhostc0,173.51.189.52,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
789D6E43-9E80-55C4-8217-141CFCFD48E0,host_simhostd0,33.159.175.99,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
B24B954E-D736-50C7-8871-1C31E7454980,host_simhosta1,58.3.70.104,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
AE0A59DD-322A-5A15-A7BC-776B847F584D,host_simhostb1,154.132.25.46,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
AF11E37D-4480-5DE2-8A95-7436F283D04C,host_simhostc1,39.45.29.111,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
E9A2DD1E-8A59-54D7-87AA-ECCAEC6106FC,host_simhostd1,32.115.19.253,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
FE57AF7C-A114-5D33-8889-B159D45C95B5,host_simhosta2,248.191.199.187,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
A4957C66-04DE-563F-A9FA-DACF4AF2C300,host_simhostb2,178.34.207.137,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
D73D0DB3-72D4-5F94-8048-BC083AB4AC88,host_simhostc2,116.7.96.46,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
F38420EE-2E09-5525-9AAD-4F4F67A5D8D9,host_simhostd2,41.74.11.147,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
8F4DED75-841E-5BD3-B98E-DDB8B0533712,host_simhosta3,205.223.79.141,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
A33700B5-AF09-5065-B8AD-A007771FF731,host_simhostc3,155.61.193.61,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
6F75E74F-9A1F-58AB-A878-EA0A2181E48C,host_simhostb3,195.175.47.12,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
66B28B4B-16E0-564E-B74B-AD09CB466887,host_simhostd3,145.191.192.40,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
0A48FFF8-B1BA-5EF8-B273-94B8ECA29C48,host_simhosta4,115.185.180.12,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
2D705531-6B36-59DA-90CA-0BE8F2DA39D7,host_simhostc4,45.31.191.90,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
1FF08533-57CD-57F4-AF17-607E088771ED,host_simhostb4,131.114.235.184,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
3F6ED0B1-4FC5-5A3D-B643-6BD3A09901A5,host_simhostd4,80.228.240.135,sim,NA,NA,39,0,0,NA,NA,NA,NA,NA,NA
DB605167-AC73-56E2-8732-4BCC122E2DC2,host_simhostb0,64.27.138.178,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
601D3AB9-C501-538D-AE5D-F3847BF93F9F,host_simhosta0,215.99.128.185,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
F7ED87C7-E217-5724-9935-F4282E6744F8,host_simhostc0,217.37.153.191,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
9B4B5C3E-F23B-5476-AAD6-272C0723FD3A,host_simhostd0,194.184.30.95,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
FFD353FD-0750-5863-9A43-E74913EC9EF6,host_simhostb1,121.63.29.216,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
C5FC530A-84C0-5AE4-AD59-26114468BC33,host_simhosta1,86.116.69.246,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
F82547E2-22FD-57E0-BE55-8B62B213624B,host_simhostd1,230.35.175.46,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
C7046C56-4FF8-5917-A6A2-AB4F8084F3F7,host_simhostc1,174.182.6.130,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
778F9EB6-91E3-5A86-B8EB-34E36339AF23,host_simhostb2,53.170.8.16,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
0AC8DBE3-E7A7-56E6-8731-27D204729587,host_simhosta2,250.167.11.199,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
F96676A9-109D-5175-B431-C19B9DC2CDB1,host_simhostd2,8.72.196.162,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
FAF9A9DB-FF04-5D6C-8D6D-D74C683780A8,host_simhostc2,154.230.138.196,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
AF0C7958-DE6B-51E0-957A-BA9712D33066,host_simhostb3,76.13.160.104,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
B0EF2428-813F-54A1-BBED-C8B16ECF166B,host_simhosta3,230.146.159.104,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
95563CA0-FA86-54F5-82BB-6A673FD74915,host_simhostd3,214.64.204.230,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
457BD9E7-D38D-57CB-836B-6CC556EFA6C5,host_simhostc3,118.253.143.122,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
5BE634DD-23F2-5549-8B84-3700ABBDAF15,host_simhostb4,26.245.59.172,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
3F0888E2-00AC-5647-82A5-F074FC081967,host_simhosta4,179.164.6.116,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
606EDA90-0B35-5830-8B3C-2D66B9F28F34,host_simhostd4,118.105.212.224,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA
6B6BFAE8-C1E1-5388-A2DB-0DB874EDC4D6,host_simhostc4,180.175.140.246,sim,NA,NA,49,0,0,NA,NA,NA,NA,NA,NA


